extensions/adobe/experience/decisioning/activity.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/activity",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decision Activity",
+  "type": "object",
+  "auditable": true,
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/record",
+    "https://ns.adobe.com/xdm/common/auditable"
+  ],
+  "description": "Entity to to control the decisioning process. It specifies the filter applied to the total inventory of options in order to narrow them down by topic/category. The activity also specifies a fallback option should the combined constraints disqualify all narrowed-down options. A decision activity ican be thought of as a type of http://schema.org/Action .",
```

Comment:
"ican"? typo. Also some white space at the end of the line needs clean up.

Reply:
fixed